### ISSUE TITLE
Can’t color toggles/ music player

### ISSUE THREAD

**Issue by nhonickman** [NoisyFlake/MagmaEvo-support#25]
I have Magma Evo 1.1.5. I have libcolorpicker 1.6.7
Installed. I am unable to color toggles or music player 
I have Prysm installed as well. I removed togglow and its plists
I am able to use your ode to pro preset for colors but toggles don’t change color

TIA

**Comment by NoisyFlake** [comment]
Why didn't you use the template to create a bug report? It's there for a reason.

Anyway, let's investigate. Is the problem that you can't choose colors in the settings or that the colors you've set in the settings simply don't apply to the Control Center? 
Does this happen with and without Prysm?

**Comment by nhonickman** [comment]
Hi. I am sorry. I thought I reported the issue correctly. If there is a template I will use that in the future.  

I really appreciate you responding to my concerns. 

I am using your posted templates and can change most items. 

I think I found the issue. In the Launchers & toggles I need to be in the disabled state to get a color to change. I had no idea about that. 
What does Regular and remove overlay change?  
  

Thank you
Neal
